plugins/pencilblue/controllers/actions/login.js

```diff
@@ -50,7 +50,7 @@ module.exports = function LoginActionControllerModule(pb) {
 
             return passport.authenticate('custom-local', (err, user) => {
                 if (!user) {
-                    return this.loginError(this.isAdminLogin, cb);
+                    return this.loginError(cb);
```

Comment:
dont care about that anymore?

Reply:
It was an error, we have access to that variable at this, so it's not necessary to pass the variable